pkg/preflights/host-preflight.yaml

```
@@ -291,3 +303,29 @@ spec:
               message: Kernel meets minimum version version of 3.10
           - fail:
               message: Kernel doesn't meet minimum version of 3.10
+    - http:
+        checkName: Can Access API
+        collectorName: http-replicated-app
+        exclude: '{{ or .IsAirgap (eq .ReplicatedAPIURL "") }}'
+        outcomes:
+          - fail:
+              when: error
+              message: 'Error connecting to {{ .ReplicatedAPIURL }}'
```

One goal of these is to highlight missing proxy configuration or a misconfigured firewall. I don't know how this will look UX-wise because it's getting long, but could we do something like this? Same for proxy service URL below.
```suggestion
              message: 'Error connecting to {{ .ReplicatedAPIURL }}. Ensure your firewall is properly configured, or use the `--http-proxy`, `--https-proxy`, and `--no-proxy` flags if there is a proxy server.'
```